src/modules/fetchingData.js

```diff
@@ -0,0 +1,52 @@
+// const dataArray = [
+//   {
+//     name: 'name:',
+//     score: '100',
+//   },
+//   {
+//     name: 'name:',
+//     score: '20',
+//   },
+//   {
+//     name: 'name:',
+//     score: '10',
+//   },
+//   {
+//     name: 'name:',
+//     score: '70',
+//   },
+//   {
+//     name: 'name:',
+//     score: '80',
+//   },
+// ];
+
+// export default dataArray;
```

Comment:
- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and readable 😉